orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/CompleteExecutionHandler.kt

```diff
@@ -84,11 +84,21 @@ class CompleteExecutionHandler(
           }
         }
       }
-      execution.pipelineConfigId?.let {
-        queue.push(StartWaitingExecutions(it, purgeQueue = !execution.isKeepWaitingPipelines))
+      log.info("Execution ${execution.id} is with ${execution.status} status and  Disabled concurrent executions is ${execution.isLimitConcurrent}")
+        if (execution.isLimitConcurrent){
+          if (execution.status != RUNNING) {
+            execution.pipelineConfigId?.let {
+              queue.push(StartWaitingExecutions(it, purgeQueue = !execution.isKeepWaitingPipelines))
+            }
+          }else {
```

Comment:
nit: Can you fix the formatting here? spaces between { } and on L96 too

Reply:
fixed the formatting